OpenEmu/ja.lproj/Localizable.strings

```diff
@@ -26,9 +26,9 @@
 
 "%@ copy" = "%@ コピー";
 
-"%@ could not be enabled because its plugin was not found." = "%@ を有効化できませんでした。プラグインが見付かりません。";
+"%@ could not be enabled because its plugin was not found." = "%@を有効化できませんでした。プラグインが見付かりません。";
 
-"%@ games will appear here. Check out %@ on how to add disc-based games." = "%@ ゲームがここに表示されます。 ディスクの入れ替えが必要なゲームは、ここ%@を確認して下さい。";
+"%@ games will appear here. Check out %@ on how to add disc-based games." = "%@のゲームがここに表示されます。 ディスクの入れ替えが必要なゲームは、こちらから%@を確認して下さい。";
```

Comment:
We don't have any spaces in Japanese, so it should be okay! If I see it's weird then I will remove from next version (but I think that would be no problem unless it will cause a problem in terms of programming language). Sorry for the late reply anyway!